spec/jobs/content_delete_event_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe ContentDeleteEventJob do
+  let(:user) { create(:user) }
+  let(:another_user) { create(:user) }
+  let(:third_user) { create(:user) }
+  let(:file_set) { create(:file_set, title: ['Hamlet'], user: user) }
+  let(:generic_work) { create(:generic_work, title: ['BethsMac'], user: user) }
+  let(:mock_time) { Time.zone.at(1) }
+  after do
+    Redis.current.keys('events:*').each { |key| Redis.current.del key }
+    Redis.current.keys('User:*').each { |key| Redis.current.del key }
+    Redis.current.keys('FileSet:*').each { |key| Redis.current.del key }
+    Redis.current.keys('GenericWork:*').each { |key| Redis.current.del key }
+  end
```

Comment:
What about putting this into a module under `spec/support` that's something like RedisCleaner.clean! and use it the same way we use ActiveFedora::Cleaner.clean!